### PR TITLE
Add tutorial images

### LINKED ISSUE #7962: Add small number of carefully curated screenshots to handbook tutorial
Anyone interested in seeing a few ideally helpful screenshots added to the handbook tutorial? E.g. 

Screenshot 2024-04-10 at 8 33 10 AM

I may like to see this done, but am also concerned we shouldn't be adding images to the main repository for a variety of reasons perhaps, but mostly size concerns.

I'm working on a Pillow demo where I follow the handbook tutorial and if nothing else they can go there. But I also wanted to rule-out-or-in the possibility of including some of these screenshots in the handbook proper. Thanks for any thoughts.

### PR DESCRIPTION
**Pull request by aclark4life**
Fixes #7962.

Changes proposed in this pull request:

 * Add tutorial images

### CODE REVIEW

File: docs/handbook/tutorial.rst [hunk 1]
```diff
@@ -444,10 +511,43 @@ Reading sequences
...
+
+    # List of image filenames
+    image_filenames = [
+        "hopper.jpg",
+        "rotated_hopper_270.jpg",
+        "rotated_hopper_180.jpg",
+        "rotated_hopper_90.jpg",
+    ]
+
+    # Open images and append them to a list
```
Comment (radarhere) [inline docs/handbook/tutorial.rst:531]:
```suggestion
    # Open images and create a list
```
Nitpick - I don't think list comprehension is 'append'ing?

File: docs/handbook/tutorial.rst [hunk 2]
```diff
@@ -358,6 +415,9 @@ Applying point transforms
     # multiply each pixel by 1.2
     out = im.point(lambda i: i * 1.2)
 
+.. image:: transformed_hopper.jpg
+    :align: center
```
Comment (radarhere) [inline docs/handbook/tutorial.rst:419]:
```python
from PIL import Image
im = Image.open("Tests/images/hopper.png")
out = im.point(lambda i: i * 1.2)
out.save("transformed_hopper.jpg")
```
gives me

transformed_hopper

Yours looks different?
transformed_hopper_pr

**Comment by radarhere** [comment]
I've pushed commits to
- fix the failing documentation build when it referenced `ImageSequence.Iterator`
- update the code for 'Copying a subrectangle from an image' to match the image you used
- remove rotated_hopper.jpg, which wasn't being referenced
- converted images to WebP where the example code didn't imply otherwise, since you were concerned about file size in https://github.com/python-pillow/Pillow/issues/7962#issue-2235492327

**Comment by aclark4life** [comment]
@radarhere Thanks for the fixes!

**Comment by aclark4life** [comment]
@hugovk OK to merge? 😇

File: docs/handbook/tutorial.rst (hunk 2 above)
Comment (hugovk) [reply docs/handbook/tutorial.rst:419]: @aclark4life Same for me, does this need fixing?

Comment (aclark4life) [reply docs/handbook/tutorial.rst:419]: Sorry, fixed in 96fa1f5dbf43fe26a9b7abd98697ae46e8165435!

**Review by radarhere — APPROVED** [review]

**Review by vinniespina — APPROVED** [review]

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**